src/hw4_Polymorphism/Competition/Pacticipants/Participant.java

```diff
@@ -0,0 +1,33 @@
+package hw4_Polymorphism.Competition.Pacticipants;
+
+import hw4_Polymorphism.Competition.Obstacle.Obstacle;
+import hw4_Polymorphism.Competition.Obstacle.RunningTrack;
+import hw4_Polymorphism.Competition.Obstacle.Wall;
+
+public abstract class Participant {
```

Comment:
why not interface?

Reply:
At first I made an interface, but then I decided that an abstract class is more convenient, since all participants have the same attributes: name, height and length limits. Do you think the interface is better?